CODE_OF_CONDUCT.md

```diff
@@ -1,40 +0,0 @@
-# Welcome to Fox's Magisk Module Manager community
-
-These guidelines set forth our expectations for our communities, not limiting but including our GitHub Repository, our Telegram groups, and other affiliated groups/communities. They really boil down to three principles:
-
-1) Don't be a dick.
-2) A dick do not be.
-3) Review rules 1 and 2.
-
-We believe in free speech, and freedom of expression, insofar that it doesn't interfere with others' right to the same. Contributors, community members, developers, and users alike are welcome here, but you must extend the same welcome to other members. So long as you can follow those three principles, you are welcome here, whether it's to hand out, lurk, or more directly contribute.
-
-## What we do not allow
-
-Absolutely no harrassment including, but not limited to, harassment based on the basis of sexuality, gender, race, or other beliefs shall be tolerated. We want everyone to feel safe and welcome here, and when some community members harass others, that is made much harder.
-
-Harassment is defined as, as per the english dictionary:
-```
-noun: harassment; plural noun: harassments
-  aggressive pressure or intimidation.
-  "they face daily harassment and assault on the streets"
-```
-
-We will also not tolerate any illegal activity, including but not limited to, copyright infringement, license infringement, threats, violence, doxxing, or any activity not legal in any team member's or developers home country.
```

Comment:
Requiring legality in the jurisdiction of contributors is both possibly mutually exclusive, but specifically can't be included in any libre software license.

It really is up to the functioning mental faculties of every contributor to take on legal repercussions in their own respective jurisdictions, and that seems more of a deterrent than this text (taking away from its more solid points) will ever do.